extension/task/utils/parseConfigFile.ts

```diff
@@ -24,11 +24,15 @@ import axios from "axios";
  * @returns {IDependabotConfig} config - the dependabot configuration
  */
 async function parseConfigFile(variables: ISharedVariables): Promise<IDependabotConfig> {
-  const possibleFilePaths = [
+  let possibleFilePaths = [
     "/.github/dependabot.yaml",
     "/.github/dependabot.yml",
   ];
 
+  if (variables.configFileLocation !== "") {
+    possibleFilePaths = [variables.configFileLocation];
```

Comment:
Should this be ``possibleFilePaths = [`"${variables.configFileLocation}"`];`` to maintain quotes around the string? Apologies, not overly familiar with TS.